README.md

```diff
@@ -1,2 +1,27 @@
-# hw01
-Repository for homework 01
+
+#__Isaac Radner__
+###_MAPSS 2019_
+####_Focus: Political Theory_
+<https://www.linkedin.com/in/isaacradner/>
+
+![It's me!](3.jpg){width=250px}
```

Comment:
Not sure if this is my problem, but I can't see the picture!

Reply:
I can see it but it hasn't been resized like @iradner intended